### PR TITLE
[Admin] Define LocationDataPointTests

### PR DESCRIPTION
**Pull request by andrewdimmer**
Working on #50 - Define LocationDataPointTests

### CODE REVIEW

File: docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.puml [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 package edu.oakland.test.admin {
     class LocationDataPointTests {
-        
+        + void validLatInIsLngOut()
```
Comment (TessaPeruzzi) [inline docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.puml]: I think this should say "validLatInIs**Lat**Out()"

File: docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.svg [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
 @startuml LocationDataPointTests
 
 package edu.oakland.test.admin {
     class LocationDataPointTests {
-        
+        + void validLatInIsLngOut()
```
Comment (TessaPeruzzi) [inline docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.svg]: I think this is also supposed to be validLatInIsLatOut()

**Review by TessaPeruzzi — CHANGES_REQUESTED** [review]
I believe the one function I commented on had a typo but other than that, all the tests look good!

File: docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.puml (hunk 1 above)
Comment (andrewdimmer) [reply docs/uml-class-diagrams/admin/test/LocationDataPointTests/LocationDataPointTests.puml]: Good catch! Thank you!

**Review by TessaPeruzzi — APPROVED** [review]
looks good!

**Review by brendanfraser597 — APPROVED** [review]

**Pull request merged**